source/read/project.txt

```diff
@@ -0,0 +1,137 @@
+.. _pymongo-project:
+
+========================
+Specify Fields To Return
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, filter, project, select
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return from a read
+operation by using a **projection**. A projection is a document that specifies
+which fields MongoDB returns from a query.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<pymongo-get-started>` guide.
+
+Projection Types
+----------------
+
+You can use a projection to specify which fields to include in a return
+document, or to specify which fields to exclude. You cannot combine inclusion and
+exclusion statements in a single projection, unless you are excluding the
+``_id`` field.
+
+Specify Fields to Include
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you use a projection to specify fields to include in the return
+document, the ``_id`` field is also included by default. All other fields are
+implicitly excluded.
+
+Use the following syntax to specify the fields to include in the result:
+
+.. code-block:: json
+
+   { "<Field Name>": 1 }
+
+The following example uses the ``find()`` method to find all restaurants with
+the ``name`` field value of ``"Emerald Pub"``. It then uses a projection to
+return only the ``name``, ``cuisine``, and ``borough`` fields in the returned
+documents.
+
+.. io-code-block::
+    
+   .. input:: /includes/project/project.py
+      :start-after: start-project-include
+      :end-before: end-project-include
+      :language: python
+      :emphasize-lines: 1
+
+   .. output::
+
+      {'_id': ObjectId('...'), 'borough': 'Manhattan', 'cuisine': 'American', 'name': 'Emerald Pub'}
+      {'_id': ObjectId('...'), 'borough': 'Queens', 'cuisine': 'American', 'name': 'Emerald Pub'}
+
+.. note::
+    
+   To remove the ``_id`` field from the return
+   document, you must :ref:`explicitly exclude it <pymongo-project-remove-id>`.
```

Comment:
Could ditch this link and say something like "as described in the following section." That's not as future-proof though

Reply:
For future-proofing and to make it more flexible I think it makes sense to keep it as a link, even though it is a bit weird to link to the next line